theano/printing.py

```diff
@@ -458,14 +456,12 @@ def process_graph(self, inputs, outputs, updates=None,
                         map(str, list(inputs) + updates.keys())))]
         else:
             strings = []
-        pprinter = self.clone_assign(lambda pstate, r: r.name is not None
-                                     and r is not current,
-                                     LeafPrinter())
+        pprinter = self.clone_assign(lambda pstate, r: r.name is not None and
+                                     r is not current, LeafPrinter())
         inv_updates = dict((b, a) for (a, b) in updates.iteritems())
         i = 1
         for node in gof.graph.io_toposort(list(inputs) + updates.keys(),
-                                          list(outputs) +
-                                          updates.values()):
+                                          list(outputs) + updates.values()):
```

Comment:
Although I think that we shouldn't run test_flake8 in python3 mode as long as we're using 2to3.